data/functions.sql

```diff
@@ -1006,28 +1006,31 @@ BEGIN
     RETURN '{}'::jsonb;
   END IF;
 
-  -- first, try the countries table
-  SELECT
-    min_label, max_label INTO min_zoom, max_zoom
-    FROM ne_10m_admin_0_countries c
-    WHERE c.wikidataid = wikidata_id;
+  -- if it's a country, only look it up in the iso and tlc tables
+  IF place_tag='country' OR place_tag='unrecognized' THEN
+      SELECT
+        min_label, max_label INTO min_zoom, max_zoom
+        FROM ne_10m_admin_0_countries_iso i
+        WHERE i.wikidataid = wikidata_id;
 
-  -- if that fails, try map_units (which contains some sub-country but super-
-  -- state level stuff such as England, Scotland and Wales).
-  IF NOT FOUND THEN
-    SELECT
-      min_label, max_label INTO min_zoom, max_zoom
-      FROM ne_10m_admin_0_map_units mu
-      WHERE mu.wikidataid = wikidata_id;
+      IF NOT FOUND THEN
+        SELECT
+            min_label, max_label INTO min_zoom, max_zoom
+        FROM ne_10m_admin_0_countries_tlc t
+        WHERE t.wikidataid = wikidata_id
+        AND fclass_tlc IN ('Admin-0 country' or 'Admin-0 dependency');
```

Comment:
This might not be the right SQL?

You will be missing the disputed areas recognized by Russia which are "Admin-0 breakaway and disputed", and you'll be missing the constituent countries of the United Kingdom (eg England) which are "Admin-1 region".

Reply:
It does yield Somaliland, Northern Cyprus, and Kosovo

Reply:
It also yields: Dhekelia Cantonment, Akrotiri Sovereign Base Area, Coral Sea Islands, Clipperton Island, Ashmore and Cartier Islands.

Reply:
This is still WIP as I haven't been able to test it so this could be wrong. 

However, the breakaways should have a `kind=unrecognized` so I'm intentionally excluding them from the where statement. They should still get a `kind:xx = country` for the viewpoints that recognize them though. 

The [data/apply-ne_country_label_recasting.sql](https://github.com/tilezen/vector-datasource/pull/2069/files/4bd88709309eb97128e023c4ac32da4216c65506#diff-ebf5e928128c82976d4f33eb55aa8c331b9e5ce90b28e13b3487ff4da7ddab35) file makes some adjustments for specific countries or features that don't conform to our policies, such as Somaliland or northern Cyprus. 

I will take a look a the England situation.

Reply:
Looks like you need to replace your 'or' with a comma on this line or split it into 2 separate fclass_tlc=<blah> clauses joined with an OR